app/services/oidc_jwt_validate.rb

```diff
@@ -22,6 +22,17 @@ def decoded_payload
 
   private
 
+  def jwk_findkey
+    finder = JWT::JWK::KeyFinder.new(jwks: jwks_loader)
+
+    # Support ADFS's use of the non-standard x5t header for holding key ID
+    lambda (header) do
```

Comment:
`lambda` has different signature then arrow function.
```suggestion
    lambda do |header|
```